mod_upload/controllers.py

```diff
@@ -222,6 +242,36 @@ def process_id(upload_id):
                     g.db.rollback()
                 # Move file
                 if db_committed:
+                    with open("issues.md", "r") as f:
+                        data = f.read()
+                    split_arr = data.split('**')
```

Comment:
I'm not too happy with splitting on that and then using it access based. I'd rather see a replace on a string instead.